docfx/doxygen2yaml.h

```diff
@@ -20,11 +20,7 @@
 #include <string>
 
 namespace docfx {
-
-struct YamlContext {
-  std::string library_root = "google/cloud/";
-  std::string parent_id;
-};
+struct YamlContext;
```

Comment:
Nit: Consider whether a forward declaration is really worth it.

Reply:
Removed.